kubescaler/scaler/aws/cluster.py

```diff
@@ -624,8 +848,15 @@ def data(self):
             "description": self.description,
         }
 
-    @retry
     def scale(self, count):
+
+        if self.eks_nodegroup:
```

Comment:
Nit - but the pattern I like here is to do one if, and then return the result of the function, and then there is no need for else. But if this is your preference we can do that. The question comes down to preference, and if there is something we want to return.

Reply:
yeah it looks good and future compatible also